texar/modules/connectors/connectors.py

```diff
@@ -550,7 +551,9 @@ class instance.
             activation_fn = get_function(self.hparams.activation_fn, fn_modules)
             output = _mlp_transform(sample, self._output_size, activation_fn)
 
-        _assert_same_size(output, self._output_size)
+            _assert_same_size(output, self._output_size)
```

Comment:
I’m thinking the logic is not very smooth, since
1) if `transform` is False and we don’t transform `sample` to `output`, then `output` equals to `sample` then `output` doesnot fit `self._output_size`;
2) if `output` is transformed anyway, then `transform` can be not necessary… then we have (output, sample) stands for (transformed(sample), sample).

Reply:
I don't understand your questions. 

pls see the [doc](https://github.com/asyml/texar/blob/70e90726bb50ba1824afc719f33280a993f51fe8/texar/modules/connectors/connectors.py#L506-L508) I marked -- if `transform` is False, then `sample` must fit `self._output_size`